era-compiler-solidity/CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+# Contribution Guidelines
+
+Hello! Thanks for your interest in joining the mission to accelerate the mass adoption of crypto for personal
```

Comment:
All this must stay at the root.
Only `src` and `Cargo.toml` go to the crate directory.